### ISSUE TITLE
Text / width limit on Progession Steppers is way too small

### ISSUE THREAD

**Issue by Solly2** [carbon-design-system/carbon#5340]
We're implementing the progressional stepper into our products and my development team have notified me that the progressional stepper has a fixed width and a character limit of 12.

I can understand why you might make the decision to make progressional steppers all the same width, but the one selected is far too small.  We cannot get the lables we need to fit into the space provided.

This issue will be compounded when we move to different languages.

**Comment by asudoh** [comment]
Hi 👋 depending on your application needs, you can always change the width by CSS override. Let us know it's not successful and we can do some assistance.
